products/error_tracking/backend/api/error_tracking.py

```diff
@@ -603,7 +605,7 @@ def bulk(self, request, **kwargs):
                 if new_status is None:
                     raise ValidationError("Invalid status")
                 for issue in issues:
-                    log_activity(
+                    _ = log_activity(
```

Comment:
Do we need to assign the result?
```suggestion
                    log_activity(
```

Reply:
Pyright gives me a warning about the unused return value 😄

I can stop doing it if you prefer, but I do personally like being explicit about unused returns